doc/api/fs.md

```diff
@@ -5113,6 +5159,7 @@ the file contents.
 [`fs.write(fd, buffer...)`]: #fs_fs_write_fd_buffer_offset_length_position_callback
 [`fs.write(fd, string...)`]: #fs_fs_write_fd_string_position_encoding_callback
 [`fs.writeFile()`]: #fs_fs_writefile_file_data_options_callback
+[`fs.writev(fd, buffers...)`]: #fs_fs_writev_fd_buffers_position_callback
```

Comment:
If https://github.com/nodejs/node/pull/25925/files#r314580192 is accepted, this will also need to be updated appropriately.